### PR TITLE
FIXED: Before this PR, only CPControl could use nib2cib-adjustment-frame.

### PR DESCRIPTION
**Pull request by didierkorthoudt**
With this PR, now all CPView can.

### CODE REVIEW

**Comment by didierkorthoudt** [comment]
milestone=1.1

**Comment by daboe01** [comment]
-#new
+Theme

File: Tools/nib2cib/NSView.j [hunk 1]
```diff
@@ -25,6 +25,7 @@
 
 @import <AppKit/CPView.j>
 
+@class Nib2Cib
```
Comment (mrcarlberg) [inline Tools/nib2cib/NSView.j]: Should not the corresponding line in NSControl.j be removed?

Comment (didierkorthoudt) [reply Tools/nib2cib/NSView.j]: You're right. Done in new commit.

**Comment by didierkorthoudt** [comment]
I've just noticed that PR #2760 deals with the same matter.
I'll merge in this PR what was complementary done in the older one.
So, please wait until a new commit.

**Comment by mrcarlberg** [comment]
Thanks! Merged!

**Pull request merged**